AjaxControlToolkit/AjaxFileUpload/AjaxFileUpload.cs

```diff
@@ -231,19 +231,22 @@ public bool AutoStartUpload
 
         protected override void OnInit(EventArgs e) {
             base.OnInit(e);
+
             if(!IsDesignMode) {
                 if(!string.IsNullOrEmpty(Page.Request.QueryString["contextkey"])
                     && 
                     Page.Request.QueryString["contextkey"] == ContextKey
                     &&
                     Page.Request.QueryString["controlID"] == ClientID)
                     IsInFileUploadPostBack = true;
+
+                ProcessRequest();
             }
         }
 
         protected override void OnLoad(EventArgs e) {
             base.OnLoad(e);
-
+            
```

Comment:
"Pull request must not change the formatting of the existing code not related to the pull request." :smile: